recipes/_configure_server.rb

```diff
@@ -5,6 +5,9 @@
 #
 # Configures the system server.conf file
 
+default_pass4symmkey = 'changeme'
+default_ssl_password = 'password'
```

Comment:
Had talked to Charlie regarding this. We decided not to do it now and we will consider doing in the future as this will include more work.